libr/bin/format/mach0/mach064_is_kernelcache.c

```diff
@@ -0,0 +1,42 @@
+static bool is_kernelcache(const ut8 *buf, ut64 length) {
+	r_return_val_if_fail (length >= sizeof (struct MACH0_(mach_header)), false);
```

Comment:
as long as the input is tainted we dont want to use an assert in here because that will crash the whole program. It is better to add an if in here